msg/DifferentialPressure.msg

```diff
@@ -1,10 +1,11 @@
-uint64 timestamp                     # time since system start (microseconds)
-uint64 timestamp_sample
+# Differential-pressure sensor
+#
+# ?
 
-uint32 device_id                     # unique device ID for the sensor that does not change between power cycles
+uint64 timestamp         # [us] Time since system start
+uint64 timestamp_sample  # [us] Timestamp of the raw data
 
-float32 differential_pressure_pa     # differential pressure reading in Pascals (may be negative)
-
-float32 temperature                  # Temperature provided by sensor in degrees Celsius, NAN if unknown
-
-uint32 error_count                   # Number of errors detected by driver
+uint32 device_id                  # [-] Unique device ID for the sensor that does not change between power cycles
+float32 differential_pressure_pa  # [Pa] Differential pressure reading in Pascals (may be negative)
+float32 temperature               # [degC] [@invalid NaN if unknown] Temperature
+uint32 error_count                # [-] Number of errors detected by driver
```

Comment:
Note, I made these grouped (no inter-line whitespace. It makes it more readable like this, and they share they quality that they carry the raw data.

Note, new unit here is "Pa" for pascals.